src/shop-detail.html

```diff
@@ -23,6 +23,19 @@
 
     <style include="shop-common-styles shop-button shop-select">
 
+      shop-button#buyNow > * {
```

Comment:
q: is `shop-button#buyNow > *` needed to increase the specificity of the selector? would  `#buyNow > button` work?